package.json

```
@@ -17,7 +17,7 @@
   "engines": {
     "node": ">=8.16.0"
   },
-  "author": "Algolia <support@algolia.com>",
+  "author": "Algolia <https://alg.li/support>",
```

between the brackets should be an email, instead you should do:

```suggestion
  "author": {
    "name": "Algolia",
    "url": "https://www.algolia.com"
  },
```